Add configuration files for Kubernetes (with HTTPs)

aegamesi opened the pull request.

kklin commented on kubernetes/clusterissuer-letsencrypt.yaml:
```diff
@@ -0,0 +1,15 @@
+    # The ACME server URL
+    server: https://acme-v01.api.letsencrypt.org/directory
+    # Email address used for ACME registration
+    email: webmaster@kelda.io
```
This doesn't exist right?

kklin replied:
Actually, lets just leave the image for now.

aegamesi commented:
Okay -- updated to use the Dockerhub container, fixed the Let's Encrypt email address, added instructions for redeployment, and cleaned up Git history

kklin commented on kubernetes/deployment.yaml:
```diff
@@ -0,0 +1,19 @@
+    spec:
+      containers:
+      - name: kelda-io
+        image: keldaio/kelda.io:latest
```
If you have some time, can you make it so dockerhub automatically tags each build with a unique tag so that we're not using latest here?

Not a big deal for our website since it doesn't change much, but I think it'd be a good exercise to see the pains in larger deployments.
